apps/notifications/templates/civic_europe_notifications/emails/notify_followers_community_award.en.email

```diff
@@ -11,10 +11,7 @@ The idea "{{ idea.title }}" has won the community award.
 {% block content %}
 The idea "{{ idea.title }}" is the
 community award winner of the {{ site.name }} {{ idea.module.name }} competition.
-The team behind the idea is now invited to the idea challenge camp, where they
-can meet other inovators and team up to increase thier changes. Afterwards they
-can transform thier idea sketch into an proposal to participate in the final
-round.
+The idea will now be included in the shortlist and receive start-up funding.
```

Comment:
Hmm, only the first part is true. The winners will then be selected from the shortlist and only the winners receive the funding. But all the effort for an email that isn't send! :roll_eyes:

Reply:
I actually took this from the welcoming e-mail after submitting an idea. There it says:

Anyone who has submitted an idea is entitled to vote for our Community Award - both applicants and co-workers. The idea with the most votes will be included in the shortlist and receive start-up funding.

Reply:
Sorry! I was very confused by all the phases don't do anything now!